precompile/allowlist/config.go

```diff
@@ -10,15 +10,15 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// Config specifies the initial set of addresses with Admin or Enabled roles.
-type Config struct {
+// AllowList specifies the initial set of addresses with Admin or Enabled roles.
+type AllowList struct {
```

Comment:
I'd prefer this to be `AllowListConfig`, since `AllowList` is more like the collection of these contract functions.